Formula/protobuf.rb

```diff
@@ -81,30 +108,8 @@ def install
     # Install editor support and examples
     doc.install "editors", "examples"
 
-    if build.with? "python"
-      # google-apputils is a build-time dependency
-      ENV.prepend_create_path "PYTHONPATH", buildpath/"homebrew/lib/python2.7/site-packages"
-
-      res = resources.map(&:name).to_set - ["gmock"]
-      res.each do |package|
-        resource(package).stage do
-          system "python", *Language::Python.setup_install_args(buildpath/"homebrew")
-        end
-      end
-      # google is a namespace package and .pth files aren't processed from
-      # PYTHONPATH
-      touch buildpath/"homebrew/lib/python2.7/site-packages/google/__init__.py"
-      chdir "python" do
-        ENV.append_to_cflags "-I#{include}"
-        ENV.append_to_cflags "-L#{lib}"
-        args = Language::Python.setup_install_args libexec
-        args << "--cpp_implementation"
-        system "python", *args
-      end
-      site_packages = "lib/python2.7/site-packages"
-      pth_contents = "import site; site.addsitedir('#{libexec/site_packages}')\n"
-      (prefix/site_packages/"homebrew-protobuf.pth").write pth_contents
-    end
+    pyinstall("python") if build.with? "python"
+    pyinstall("python3") if build.with? "python3"
```

Comment:
Use `Language::Python.each_python(build) do |python, python_version|` or `python_versions.each` over some array instead of creating a new function.